shuup/admin/static_src/base/scss/shuup/auth.scss

```diff
@@ -1,11 +1,18 @@
+.auth {
+    @media (max-width: 767.98px) {
```

Comment:
Would it be possible to use breakpoints on these three `@media` definitions?

Reply:
Applies to the other 2 places in this file too.